README.md

```diff
@@ -32,22 +33,237 @@ scheduled_task { 'Run Notepad':
 
 ## Usage
 
-See the [Puppet resource documentation](https://puppet.com/docs/puppet/latest/types/scheduled_task.html) for more information.
+One common use is to kick off a script, either once or on a regular cadence.
```

Comment:
nit: Don't say 'One common' and then list multiple options.  Either "Common uses cases are ..." or drop the options

Reply:
Ah, this is a language fail on my part. The _use case_ is to kick off a script, the _implementation details_ are once-only or repeating. At least, that's how I meant it in my head. Will adjust for clarity, thanks!